Templates/AzureDevOpsPipeline/README.md

```diff
@@ -47,18 +70,27 @@ Variable | Description
   zosSSHKnownHost                      | The known host entry for secure shell connections. [See Notes](#obtaining-the-known-host-entry-for-secure-shell-connections).
   zosSSHPrivateKeySecureFile           | Reference to uploaded Private SSH Key in ADO Pipeline/Libary/SecureFile that is installed for sftp processes. [See Notes](#upload-private-ssh-key-as-secure-file)
   pipelineWorkspace                    | Root directory on z/OS Unix System services to perform builds. E.g. `/u/ado/workspace`
-  wdEnvironmentFile                    | The Wazi Deploy environment file for the deployment into the lowest stage
   zosHostname                          | zOS - Host name (or Host IP address) for SFTP connection
   zosSFTPUser                          | zOS - Host user for SFTP connection
   azureArtifactFeedID                  | Feed ID of the Azure artifact for publishing the package (when publishing to Azure DevOps Artifacts)
   azureArtifactVersionOption           | Azure artifact version option (when publishing to Azure DevOps Artifacts)
 
+### Supplied Azure pipeline templates 
+
+The directory [templates](templates/) contains additional [Azure pipeline templates](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops&pivots=templates-includes#parameters-to-select-a-template-at-runtime) that implement actions that are leveraged by the core pipeline template to provide a central control over the tagging and release deployment process. Please review the templates with your Azure administrator and adjust the reference within the [azure-pipelines.yml](azure-pipelines.yml#L32) file.
```

Comment:
Could we rephrase to something like:
The directory [templates](templates/) contains additional [Azure pipeline templates](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops&pivots=templates-includes#parameters-to-select-a-template-at-runtime) that implement actions used in the main pipeline template. These actions help to keep a central control over the tagging and release deployment process.